package/MDAnalysis/transformations/positionaveraging.py

```diff
@@ -20,34 +39,73 @@
 
 
 
-class PositionAverager:  
+class PositionAverager(object):  
     """
-    
-    Averages the coordinates of a given timestep with the N previous frames. 
-    For frames < N, the average of the frames iterated up to that point will be
-    returned.  
+   
+    Averages the coordinates of a given timestep so that the coordinates
+    of the given atomgroup correspond to the average of the positions 
+    of the N previous frames. 
+    For frames < N, the average of the frames iterated up to that point will
+    be returned.  
     
     Example
     -------
+
+    e.g. Average the coordinates of a given AtomGroup over the course of
+    the previous N frames. For N=3, the output will correspond to the
+    average of the coordinates over the last 3 frames.
+    When check_reset=True, the averager will be reset once the iteration is
+    complete, or if the frames iterated are not sequential.
+
+    .. code-block:: python
+        
+        N=3
+        transformation = PositionAverager(N, check_reset=True)
+        u.trajectory.add_transformations(transformation)   
+        for ts in u.trajectory:
+            print(ts.positions)
     
-    u.trajectory.add_transformations(PositionAverager(N, check_reset=True))    
+    e.g. When check_reset=False, the average of coordinates from non
+    sequential timesteps can also be computed. However, the averager must be
+    manually reset before restarting an iteration.
+           
+    .. code-block:: python
+        
+        N=3
+        transformation = PositionAverager(N, check_reset=False)
+        u.trajectory.add_transformations(transformation)
+        frames = [0, 7, 1, 6]        
+        transformation.resetarrays()
+        for ts in u.trajectory[frames]:
+            print(ts.positions)
     
+    e.g. For frames < N, the average is calculated with the frames iterated up
+    to that point and thus will not follow the same behaviour as for the
+    frames > N. This behaviour can be followed by comparing current_avg to 
+    avg_frames.For N=3, the first 2 frames should be skipped.
+
+    .. code-block:: python
+        
+        N=3
+        transformation = PositionAverager(N, check_reset=True)
+        u.trajectory.add_transformations(transformation) 
+        for ts in u.trajectory:
+            if transformation.current_avg == transformation.avg_frames:
+                print(ts.positions)
     
     Parameters
     ----------
     avg_frames: int
-    Determines the number of frames to be used for the position averaging. 
-    
-    
+        Determines the number of frames to be used for the position averaging. 
     check_reset: bool, optional
-    If 'True', position averaging will be reset, and a warning raised,
-    if the trajectory iteration direction changes. If 'False', position
-    averaging will not reset, regardless of the iteration.
+        If 'True', position averaging will be reset, and a warning raised,
```

Comment:
Use single backtics when referencing kwargs and double backtics for code and ``True`` and ``False`` so that they are typeset consistently.